securedrop/dockerfiles/xenial/python3/Dockerfile

```diff
@@ -18,32 +18,43 @@ RUN apt-get update && apt-get install -y paxctl && \
 
 RUN gem install sass -v 3.4.23
 
-ENV FF_ESR_VER 60.6.1esr
-RUN curl -LO https://ftp.mozilla.org/pub/firefox/releases/${FF_ESR_VER}/linux-x86_64/en-US/firefox-${FF_ESR_VER}.tar.bz2 && \
-    curl -LO https://ftp.mozilla.org/pub/firefox/releases/${FF_ESR_VER}/linux-x86_64/en-US/firefox-${FF_ESR_VER}.tar.bz2.asc && \
-    gpg --recv-key --keyserver gpg.mozilla.org 0x61B7B526D98F0353 && \
-    gpg --verify firefox-${FF_ESR_VER}.tar.bz2.asc && \
-    tar xjf firefox-*.tar.bz2 && \
-    mv firefox /usr/bin && \
-    paxctl -cm /usr/bin/firefox/firefox
+# Current versions of the test browser software. Tor Browser is based
+# on a specific version of Firefox, noted in Help > About Tor Browser.
+# Ideally we'll keep those in sync.
+ENV FF_VERSION 68.12.0esr
+ENV GECKODRIVER_VERSION v0.27.0
+ENV TBB_VERSION 9.5.4
 
-COPY ./tor_project_public.pub /opt/
+# Import Tor release signing key
+ENV TOR_RELEASE_KEY_FINGERPRINT "EF6E286DDA85EA2A4BA7DE684E2C6E8793298290"
+RUN curl -s https://openpgpkey.torproject.org/.well-known/openpgpkey/torproject.org/hu/kounek7zrdx745qydx6p59t9mqjpuhdf | gpg2 --import -
```

Comment:
Will this URL change when the key is updated? or is it based on the UID?

Reply:
:shrug:

Reply:
My understanding of the WKD spec is that it's based on the local part of the UID, so it should be stable.

> OpenPGP defines its User IDs, and thus the mail address, as UTF-8 strings. To help with the common pattern of using capitalized names (e.g. "Joe.Doe@example.org") for mail addresses, and under the premise that almost all MTAs treat the local-part case-insensitive and that the domain-part is required to be compared case-insensitive anyway, all upper-case ASCII characters in a User ID are mapped to lowercase. Non-ASCII characters are not changed.
>
> The so mapped local-part is hashed using the SHA-1 algorithm. The resulting 160 bit digest is encoded using the Z-Base-32 method as described in [RFC6189], section 5.1.6. The resulting string has a fixed length of 32 octets. 

https://tools.ietf.org/id/draft-koch-openpgp-webkey-service-07.html